lms/djangoapps/completion/waffle.py

```diff
@@ -71,7 +70,8 @@ def visual_progress_enabled(course_key):
     if not waffle().is_enabled(ENABLE_COMPLETION_TRACKING):
         return
 
-    if not site_configuration_enabled():
+    current_site = get_current_site()
+    if not current_site.configuration.get_value(ENABLE_SITE_VISUAL_PROGRESS, False):
         return
```

Comment:
@schenedx We still need the SiteConfiguration exception handler, there are none by default.

Reply:
Good point. Updated.